tools/docker/docker-compose-dev.yaml

```diff
@@ -44,10 +44,8 @@ x-environment: &common-env
   EDA_SERVER_UUID: edgecafe-beef-feed-fade-decadeedgecafe
   EDA_PG_NOTIFY_DSN: "host=host.containers.internal port=5432 dbname=eda user=postgres password=secret"
   EDA_PG_NOTIFY_DSN_SERVER: "host=postgres port=5432 dbname=eda user=postgres password=secret"
-  EDA_EVENT_STREAM_BASE_URL: ${EDA_EVENT_STREAM_BASE_URL:-https://localhost/edgecafe-beef-feed-fade-decadeedgecafe/}
```

Comment:
can we use `EDA_UI_PORT` instead of hard-coded 8443?

Reply:
@hsong-rh You beat me to it. :-)

Reply:
@hsong-rh @jshimkus-rh I don't think we can nest one setting inside another setting.